ci/pipeline.yml

```diff
@@ -292,11 +291,9 @@ resources:
 - name: common-production
   type: cg-common
```

Comment:
Need to change the resource type too. After that, we should be able to drop the `cg-common` resource type from the pipeline.

Reply:
Thanks!  changed.